Functions/Public/New-GitHubOrganization.ps1

```diff
@@ -0,0 +1,61 @@
+function New-GitHubOrganization {
+    <#
+    .Synopsis
+    Creates a new GitHub Organization.
+
+    .Parameter Name
+    Required. The name of the new GitHub Organization to create.
+
+    .Parameter Administrator
+    The GitHub username of the account manages the GitHub organization.
+
+    .Parameter DisplayName
+    The organization's display name.
+
+    .Link
+    https://twitter.com/michaelsainz
+    https://developer.github.com/enterprise/2.17/v3/enterprise-admin/orgs/#create-an-organization
+    #>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory, Position = 0)]
+        [Alias('Organization', 'Org')]
+        [string[]]$Name,
+
+        [Parameter(Mandatory, Position = 1)]
+        [Alias('Admin')]
+        [string]$Administrator,
+
+        [Parameter()]
+        [string]$Description,
+
+        [Parameter(Position = 3)]
+        [Security.SecureString]$Token,
+
+        [Parameter(Mandatory)]
+        [Alias('ComputerName')]
+        [string]$HostName
+    )
+    
+    Begin {
+        Write-Debug -Message 'Entered Function: New-GitHubOrganization'
+
+        $Uri = "https://$HostName/api/v3/admin/organizations"
```

Comment:
If we introduce GHE support, cmdlets should work for both GHE and github.com. This URL wouldn't work for github.com, because the base URL does not have `/api` in it.

I would propose a `BaseUrl` parameter for all cmdlets that is resolved in `Invoke-GitHubApi`.

Reply:
Absolutely agree, @felixfbecker. The saving issue with New-GitHubOrganization is that the endpoint is only available to GHES but the other two cmdlets in this PR would need that work. 👍

Reply:
Added `-HostName` property and ensured that it worked for both GHEC and GHES.